RecoTracker/IterativeTracking/python/TobTecStep_cff.py

```diff
@@ -361,22 +364,22 @@
 tobTecStep.inputClassifiers=['tobTecStepClassifier1','tobTecStepClassifier2']
 
 from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
-
-#LWTNN selector
-from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
-from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-trackingPhase1.toReplaceWith(tobTecStep, TrackLwtnnClassifier.clone(
+trackingPhase1.toReplaceWith(tobTecStep, tobTecStepClassifier1.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorTobTecStep_Phase1'),
      src = 'tobTecStepTracks',
-     qualityCuts = [-0.4, -0.25, -0.1]
+     qualityCuts = [-0.6,-0.45,-0.3]
 ))
 (trackingPhase1 & fastSim).toModify(tobTecStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
```

Comment:
this is not needed: the fastsim value is inherited from L354 and this line can be removed

Reply:
Removed the redundant line